scripts/run_overnight_bench_smoke.py

```
@@ -0,0 +1,206 @@
+
+    async def run():
+        set_poll_running(sid, True)
+        results["steps"].append({"phase": "60s", "durations": await cycles(2, 60)})
+        results["steps"].append({"phase": "30s", "durations": await cycles(2, 30)})
+        upsert_station({**(get_station(sid) or {}), "id": sid, "poll_interval_seconds": 60})
+        results["restored_60s"] = True
+
+    try:
+        asyncio.run(run())
+        results["ok"] = True
+    except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)